src/components/checkbox/Checkbox.vue

```diff
@@ -2,11 +2,14 @@
     <label class="checkbox"
         :class="{ 'is-disabled': disabled }"
         ref="label"
+        :for="nativeId"
         :disabled="disabled"
         :tabindex="disabled ? false : 0"
         @keydown.prevent.enter.space="$refs.label.click()">
         <input v-model="newValue"
             type="checkbox"
+            v-bind="$attrs"
+            :id="nativeId"
```

Comment:
You can remove it because `v-bind="$attrs"` add custom attribute and i think `:for` it's not required if checkbox is bounded by `'<b-field>`.